apps/fluent-tester/package.json

```diff
@@ -30,6 +30,7 @@
     "@fluentui-react-native/experimental-avatar": ">=0.7.1 <1.0.0",
     "@fluentui-react-native/experimental-button": "0.5.1",
     "@fluentui-react-native/experimental-native-button": ">=0.5.3 <1.0.0",
+    "@fluentui-react-native/experimental-shimmer": "0.1.0",
```

Comment:
So I know I told you to move this to experimental.. but I wonder if this point we can just stop calling stuff made with the experimental framework "experimental" and your shimmer could just be the newest update to the existing "@fluentui-react-native/shimmer" package...

Reply:
I prefer keeping the prefix for consistency and clarity... maybe folks from CXE can weigh in

Reply:
Ah, that's fair. If that's the case, can we not delete the original shimmer, keep the experimental shimmer, and delete the original shimmer in a followup?

I don't feel too strongly about this, so not blocking.

Reply:
Reason partially being, I don't know how deleting a package from a monorepo behaves with NPM. Especially if we are probably going to bring it back up later when `experimental-shimmer` just becomes `shimmer`

Reply:
I think it was mostly stuff like the CHANGELOG.JSON carrying over, or the API changing a bunch between 2 versions of shimmer.

If we deleted old, then told clients to use experimental-shimmer, then eventually moved `experimental-shimmer` to just be `shimmer` I wanted to be sure that didn't confuse people or the NPM registry. It sounds like it won't confuse the latter, and I'm pretty sure nobody consumed the old shimmer so we should be good on both fronts.

That being said... how do we feel about dropping the `experimental` prefix altogether? It seems like the experimental compose framework is just... the framework now, and is even named as such ( `@fluentui-react-native/framework`). There isn't a precedent for new controls using the new framework to be in `packages/components` instead of `packages/experimental` (besides the old shimmer, which I wrote with the new framework) but... we can set one?

Reply:
I also just remembered: I made the original Shimmer component with the new compose framework but put it in `packages/components` after asking @FalseLobster where to put it. and I only put `experimental-avatar` in `packages/experimental` to keep it distinguished from PersonaCoin (I still want to rectify those both together someday..). 
Others can chime in but I think we're at the point that we don't need to put new stuff in `packages/experimental` just because it's using the compose v2 framework?